### PR TITLE
operator hyperfoil-bundle (0.21.0)

### PR DESCRIPTION
**Pull request by rvansa**
### Updates to existing Operators

* [x] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [x] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [x] Have you tested an update to your Operator when deployed via OLM?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [x] Modify more than one operator
* [x] Modify an Operator you don't own
* [x] Rename an operator - please remove and add with a different name instead
* [x] Modify any files outside the above mentioned folders
* [x] Contain more than one commit. **Please squash your commits.**

### CODE REVIEW

File: operators/hyperfoil-bundle/0.21.0/hyperfoil-operator.v0.21.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+  maintainers:
+  - email: rvansa@redhat.com
+    name: Radim Vansa
+  - email: spederse@redhat.com
+    name: Ståle W. Pedersen
+  maturity: alpha
+  provider:
+    name: Red Hat, Inc.
+  version: 0.21.0
+  replaces: 0.15.1
```
Comment (J0zi) [inline operators/hyperfoil-bundle/0.21.0/hyperfoil-operator.v0.21.0.clusterserviceversion.yaml:310]: @rvansa please use `replaces: hyperfoil-operator.v0.15.1` here.

Comment (rvansa) [reply operators/hyperfoil-bundle/0.21.0/hyperfoil-operator.v0.21.0.clusterserviceversion.yaml:310]: Thanks!

**Comment by mvalarh** [comment]
@rvansa please fix https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/1806#issuecomment-1303179822

**Comment by J0zi** [comment]
@rvansa any update?

**Comment by J0zi** [comment]
@rvansa any update on this?

**Comment by rvansa** [comment]
Damned, sorry, got a full TODO. I'll close and reopen when this is fixed.

**Comment by rvansa** [comment]
I've added `packagemanifestClusterVersionLabel: auto` to `ci.yaml` if that's the bit pipelines were missing... Trying to reopen now.

**Pull request merged**